spec/models/user/human_spec.rb

```diff
@@ -27,6 +27,19 @@
       expect(user.errors.keys).to eq([:username])
     end
 
+    context 'personal team' do
+      it 'creates personal team' do
+        user = users(:alice).dup
+        user.username = "Alice2"
+        user.save!
+        personal_team = user.personal_team
+        expect(user.personal_team).to be_present
+        expect(personal_team.members.include?(user))
```

Comment:
maybe use helper on team?

https://github.com/puzzle/cryptopus/blob/feature-559-personal_team/app/models/team.rb#L61
```suggestion
        expect(personal_team.teammember?(user)).to eq(true)
```
expect(